tasks/travis-pr-title-lint.js

```diff
@@ -152,12 +154,30 @@ module.exports = function(grunt) {
         return;
       }
 
+      var config = path.join(__dirname, '..', '.conventional-changelog-lintrc');
+      var checkLintConfig = Promise.all([
+        fs.stat(config), fs.access(config, fs.R_OK),
+      ])
+        .then(function(results) {
+          const configStat = results[0];
+
+          if (!configStat.isFile()) {
+            throw new Error(
+              `Config file "${config}" found, but it should be a regular file`
+            );
+          }
+        })
+        .catch(function(err) {
+          throw new Error(`Unable to load changelog lint config: ${err.stack}`);
+        });
+
       Promise.all([
+        checkLintConfig,
         getPreset('angular'),
-        getConfiguration('.conventional-changelog-lintrc'),
+        getConfiguration(),
```

Comment:
hmm, why do you no longer need to pass in the file? Is it because we can assume it will be auto-detected? I thought it would be: `getConfiguration(config)`